### ISSUE TITLE
[DOCS] Revise changelogs for v12.4

### ISSUE THREAD

**Issue by TYPO3IncTeam** [TYPO3-Documentation/Changelog-To-Doc#476]
:information_source: View this commit [on Github](https://github.com/TYPO3/typo3/commit/e864e328ae38f487a984ed2519ee648321e1bfba)
:busts_in_silhouette: Authored by Chris Müller typo3@krue.ml
:heavy_check_mark: Merged by Lina Wolf 112@linawolf.de

## Commit message

[DOCS] Revise changelogs for v12.4

Resolves: #100753
Releases: main
Change-Id: Ia3d98fad1be54ad3056b746c436556f33d2a7818
Reviewed-on: https://review.typo3.org/c/Packages/TYPO3.CMS/+/78842
Tested-by: Georg Ringer <georg.ringer@gmail.com>
Tested-by: Lina Wolf <112@linawolf.de>
Tested-by: core-ci <typo3@b13.com>
Reviewed-by: Georg Ringer <georg.ringer@gmail.com>
Reviewed-by: Lina Wolf <112@linawolf.de>

## :heavy_division_sign: Modified files

<details>
<summary>12.4/Deprecation-100173-VariousMethodsAndPropertiesInUserAuthenticationClassesNowInternal.rst</summary>

```diff
@@ -13,22 +13,22 @@ Description
 
 Various methods and properties within the main classes regarding frontend
 user and backend user (:php:`$GLOBALS[BE_USER]`) authentication handling
-have been either been marked as internal or have been deprecated for usage
+have been either marked as internal or have been deprecated for usage
 outside of the classes.
 
 This is due to the further refactorings and decoupling work, as subclasses of
 :php:`AbstractUserAuthentication` deal with many more functionality nowadays,
 and therefore have been moved to service classes. The tight coupling of these
-classes e.g. the database fields, or login form field names are now marked as
+classes, for example, the database fields, or login form field names are now marked as
 internal, as these properties should not be modified from the outside scope.
 
 Instead, functionality like :ref:`PSR-14 events <t3coreapi:EventDispatcher>` or
 :ref:`Authentication Services <t3coreapi:authentication>` should influence the
 authentication and authorization workflow.
 
 The following properties and methods are now marked as internal in all
-user authetication related classes (extending
-:php:`TYPO3\CMS\Core\Authentication\AbstractUserAuthentication`):
+user authentication related classes (extending
+:php:`\TYPO3\CMS\Core\Authentication\AbstractUserAuthentication`):
 
 * :php:`lastLogin_column`
 * :php:`formfield_uname`
@@ -43,9 +43,9 @@ user authetication related classes (extending
 * :php:`unpack_uc()`
 * :php:`appendCookieToResponse()`
 
-Additionally are the following properties of the
-:php:`TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication`
-implementation marked as internal:
+Additionally, the following properties of the
+:php:`\TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication`
+implementation are marked as internal:
 
 * :php:`formfield_permanent`
 * :php:`is_permanent`
@@ -55,7 +55,7 @@ Impact
 ======
 
 The affected properties and methods have been marked as `@internal` and set to
-:php:`protected`. With an additional trait, it's still possible to access them
+:php:`protected`. With an additional trait, it is still possible to access them
 in TYPO3 v12. In case third-party extensions call them, a PHP deprecation
 warning is thrown.
 
@@ -72,7 +72,7 @@ Migration
 
 Depending on the specific requirements, it is recommended to use
 :ref:`PSR-14 events <t3coreapi:EventDispatcher>` or
-:ref:`Authentication Services <t3coreapi:authentication>` to modify behaviour
-of the Authentication classes.
+:ref:`authentication services <t3coreapi:authentication>` to modify behaviour
+of the authentication classes.
 
 .. index:: Backend, Frontend, PHP-API, FullyScanned, ext:core
```

</details>

<details>
<summary>12.4/Deprecation-100335-TCAConfigMM_insert_fields.rst</summary>

```diff
@@ -59,9 +59,9 @@ definition.
 
 Note on the related option :php:`MM_match_fields`: This is important when an
 MM relation allows multiple "right" sides. In the example above, when a category
-is added to a tt_content record using the :sql:`categories` field, and when editing
-this relation from the "right" side (editing a tt_content record), then this option
-is used to select only relations for this tt_content/categories combination. The
+is added to a :sql:`tt_content` record using the :sql:`categories` field, and when editing
+this relation from the "right" side (editing a :sql:`tt_content` record), then this option
+is used to select only relations for this :sql:`tt_content.categories` combination. The
 TCA column :sql:`categories` thus uses :sql:`MM_match_fields` to restrict the
 query. Note :sql:`MM_match_fields` is *not* set for the "left-side" :sql:`sys_category`
 :sql:`items` fields, this would indicate a TCA misconfiguration.
@@ -70,7 +70,7 @@ Various extensions in the wild did not get these details right, and often simply
 set *both* :php:`MM_insert_fields` and :php:`MM_match_fields` to the same values.
 Removing :php:`MM_insert_fields` helps reducing confusion and simplifies this
 construct a bit. Affected extensions can simply remove the :php:`MM_insert_fields`
-configuration and keep the :php:`MM_match_fields`. Note the core strives to further
+configuration and keep the :php:`MM_match_fields`. Note the Core strives to further
 simplify these options and :php:`MM_match_fields` may become fully obsolete in the
 future as well.
 
```

</details>

<details>
<summary>12.4/Deprecation-100349-TypoScriptLoginUserAndUsergroupConditions.rst</summary>

```diff
@@ -22,7 +22,7 @@ conditions based on the variables :typoscript:`frontend.user` and
 Impact
 ======
 
-Using the old conditions in Frontend TypoScript or TSconfig triggers a
+Using the old conditions in frontend TypoScript or TSconfig triggers a
 deprecation level log entry in TYPO3 v12 and will stop working with
 TYPO3 v13.
 
@@ -38,10 +38,10 @@ instances can be adapted quite easily.
 Migration
 =========
 
-There is a rather straight forward migration path. In general, switch to
+There is a rather straightforward migration path. In general, switch to
 either :typoscript:`frontend.user` to test for frontend user state
-(available in Frontend TypoScript), or to :typoscript:`backend.user` (available
-in Frontend TypoScript and TSconfig).
+(available in frontend TypoScript), or to :typoscript:`backend.user` (available
+in frontend TypoScript and TSconfig).
 
 Note the transition can be done in existing TYPO3 v11 projects already.
 
```

</details>

<details>
<summary>12.4/Deprecation-100355-DeprecateMethodsInPasswordChangeEventInExtfelogin.rst</summary>

```diff
@@ -11,7 +11,7 @@ See :issue:`100355`
 Description
 ===========
 
-The following methods in the PSR-14 Event :php:`PasswordChangeEvent` of
+The following methods in the PSR-14 event :php:`PasswordChangeEvent` of
 ext:felogin have been marked as deprecated and should not be used any more:
 
 * :php:`setAsInvalid()`
@@ -31,7 +31,7 @@ message. The functionality is kept in TYPO3 v12 but will be removed in v13.
 Affected installations
 ======================
 
-Instances who use the PSR-14 Event :php:`PasswordChangeEvent` for password
+Instances who use the PSR-14 event :php:`PasswordChangeEvent` for password
 validation and who use one of the deprecated methods.
 
 The extension scanner reports usages as a weak match.
```

</details>

<details>
<summary>12.4/Deprecation-100405-PropertyTypoScriptFrontendController-type.rst</summary>

```diff
@@ -11,13 +11,13 @@ See :issue:`100405`
 Description
 ===========
 
-The public property :php:`type` of the main class in TYPO3 Frontend
-:php:`TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController` has been
+The public property :php:`type` of the main class in TYPO3 frontend
+:php:`\TYPO3\CMS\Frontend\Controller\TypoScriptFrontendController` has been
 marked as internal, as it should not be used outside of this PHP class anymore
 in the future.
 
 This is part of the overall part to reduce dependencies on this PHP class, as
-it isn't always available in TYPO3 Frontend.
+it is not always available in TYPO3 frontend.
 
 
 Impact
@@ -46,13 +46,23 @@ the request object is not available.
 
 Within TypoScript, conditions and getData properties need to be adapted:
 
-Before: :typoscript:`[getTSFE() && getTSFE().type == 13]`
-After: :typoscript:`[request.getPageArguments()?.getPageType() == 13]`
+..  code-block:: typoscript
 
-In Typoscript getData attributes:
+    # Before
+    [getTSFE() && getTSFE().type == 13]
 
-Before: :typoscript:`page.10.data = TSFE:type`
-After: :typoscript:`page.10.data = request:routing|pageType`
+    # After
+    [request.getPageArguments()?.getPageType() == 13]
+
+In TypoScript getData attributes:
+
+..  code-block:: typoscript
+
+    # Before
+    page.10.data = TSFE:type
+
+    # After
+    page.10.data = request:routing|pageType
 
 
 .. index:: Frontend, TypoScript, FullyScanned, ext:frontend
```

</details>

<details>
<summary>12.4/Deprecation-100454-LegacyTreeImplementations.rst</summary>

```diff
@@ -12,18 +12,19 @@ Description
 ===========
 
 Due to many refactorings in TYPO3's tree implementations in the past versions,
-many implementations and functionality of the legacy rendering :php:`AbstractTreeView`
+many implementations and functionality of the legacy rendering :php:`\TYPO3\CMS\Backend\Tree\AbstractTreeView`
 is not needed anymore.
 
 The following PHP classes are not in use anymore and have been marked as deprecated:
-* :php:`TYPO3\CMS\Backend\Tree\View\BrowseTreeView`
-* :php:`TYPO3\CMS\Backend\Tree\View\ElementBrowserPageTreeView`
+
+* :php:`\TYPO3\CMS\Backend\Tree\View\BrowseTreeView`
+* :php:`\TYPO3\CMS\Backend\Tree\View\ElementBrowserPageTreeView`
 
 The base class is still available, but discouraged to be used or extended,
 even though TYPO3 still uses this in a few places.
 
 The following properties and methods within the base class
-:php:`AbstractTreeView` have either been marked as deprectaed or
+:php:`AbstractTreeView` have either been marked as deprecated or
 declared as internal:
 
 * :php:`AbstractTreeView->thisScript`
@@ -51,7 +52,7 @@ Impact
 
 Instantiating the deprecated classes or calling the deprecated methods will
 trigger a PHP deprecation warning, except for
-:php:`AbstractTreeView->getThisScript()`, which is still used internaly by
+:php:`AbstractTreeView->getThisScript()`, which is still used internally by
 deprecated code.
 
 The Extension Scanner will find those usages and additionally also reports
@@ -70,7 +71,7 @@ Migration
 =========
 
 It is recommended to avoid generating the markup directly in PHP. Instead use
-one of various other tree functionalities (see e.g. PageTree implementations)
+one of various other tree functionalities (for example, see PageTree implementations)
 in PHP and render trees via web components or Fluid.
 
 .. index:: Backend, PHP-API, FullyScanned, ext:backend
```

</details>

<details>
<summary>12.4/Deprecation-100459-BackendUtilitygetRecordToolTip.rst</summary>

```diff
@@ -11,7 +11,7 @@ See :issue:`100459`
 Description
 ===========
 
-The method :php:`TYPO3\CMS\Backend\Utility\BackendUtility::getRecordToolTip()`
+The method :php:`\TYPO3\CMS\Backend\Utility\BackendUtility::getRecordToolTip()`
 has been marked as deprecated.
 
 
@@ -37,13 +37,13 @@ with a "title" attribute for the markup, the replacement is straightforward:
 
 Before:
 
-.. code-block:: php
+..  code-block:: php
 
     $link = '<a href="..." ' . BackendUtility::getRecordToolTip(...) . '>my link</a>';
 
 After:
 
-.. code-block:: php
+..  code-block:: php
 
     $link = '<a href="..." title="' . BackendUtility::getRecordIconAltText(...) . '">my link</a>';
 
```

</details>

<details>
<summary>12.4/Deprecation-100581-AvoidConstructorArgumentInFormDataCompiler.rst</summary>

```diff
@@ -38,14 +38,19 @@ second argument to :php:`compile()` and will not be optional anymore.
 Migration
 =========
 
-.. code-block:: php
+..  code-block:: php
 
     // before
-    $formDataCompiler = GeneralUtility::makeInstance(FormDataCompiler::class, GeneralUtility::makeInstance(MyDataGroup::class));
+    $formDataCompiler = GeneralUtility::makeInstance(
+        FormDataCompiler::class, GeneralUtility::makeInstance(MyDataGroup::class)
+    );
     $formData = $formDataCompiler->compile($myFormDataCompilerInput);
 
     // after
     $formDataCompiler = GeneralUtility::makeInstance(FormDataCompiler::class);
-    $formData = $formDataCompiler->compile($myFormDataCompilerInput, GeneralUtility::makeInstance(MyDataGroup::class));
+    $formData = $formDataCompiler->compile(
+        $myFormDataCompilerInput,
+        GeneralUtility::makeInstance(MyDataGroup::class)
+    );
 
 .. index:: Backend, PHP-API, NotScanned, ext:backend
```

</details>

<details>
<summary>12.4/Deprecation-100584-GeneralUtilitylinkThisScript.rst</summary>

```diff
@@ -14,10 +14,10 @@ Description
 The method :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::linkThisScript()`
 has been marked as deprecated and should not be used any longer.
 
-The method uses the super global :php:`$_GET` which should be avoided by
-retrieving data from the PSR-7 :php:`ServerRequestInterface` instead.
+The method uses the super global :php:`$_GET` which should be avoided. Instead,
+data should be retrieved via the PSR-7 :php:`ServerRequestInterface`.
 
-Controllers should typically create urls using the :php:`\TYPO3\CMS\Backend\Routing\UriBuilder`.
+Controllers should typically create URLs using the :php:`\TYPO3\CMS\Backend\Routing\UriBuilder`.
 
 
 Impact
@@ -30,7 +30,7 @@ method will be removed with TYPO3 v13.
 Affected installations
 ======================
 
-The methods was typically used in backend context: Extensions with own
+The method was typically used in backend context: Extensions with own
 backend modules may be affected. The extension scanner finds usages
 with a strong match.
 
```

</details>

<details>
<summary>12.4/Deprecation-100587-DeprecateFormEngineAdditionalJavaScriptPostAndCustomEvalInlineJavaScript.rst</summary>

```diff
@@ -12,7 +12,7 @@ Description
 ===========
 
 The result property `additionalJavaScriptPost` of the form engine result array
-is deprecated. It was used for instance in custom eval definitions, that provided
+is deprecated. It was used, for instance, in custom eval definitions, that provided
 inline JavaScript (configured via :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['tce']['formevals']`).
 
 
@@ -45,10 +45,11 @@ Example
 
 Deprecated custom eval implementation:
 
-.. code-block: php
+..  code-block:: php
 
     <?php
     namespace TYPO3\CMS\Redirects\Evaluation;
+
     class SourceHost
     {
         public function returnFieldJS(): string
@@ -66,10 +67,11 @@ Deprecated custom eval implementation:
 Migrated custom eval implementation (JavaScript is now bundled in module
 :js:`@typo3/redirects/form-engine-evaluation.js`):
 
-.. code-block: php
+..  code-block:: php
 
     <?php
     namespace TYPO3\CMS\Redirects\Evaluation;
+
     class SourceHost
     {
         public function returnFieldJS(): JavaScriptModuleInstruction
```

</details>

<details>
<summary>12.4/Deprecation-100596-GeneralUtility_GET.rst</summary>

```diff
@@ -44,7 +44,7 @@ Migration
 =========
 
 :php:`GeneralUtility::_GET()` is a helper method that retrieves
-incoming HTTP `GET` query arguments returns the value.
+incoming HTTP `GET` query arguments and returns the value.
 
 The same result can be achieved by retrieving arguments from the request object.
 An instance of the PSR-7 :php:`ServerRequestInterface` is handed over to
```

</details>

<details>
<summary>12.4/Deprecation-100597-BackendUtilityMethodsGetThumbnailUrlAndGetLinkToDataHandlerAction.rst</summary>

```diff
@@ -11,8 +11,8 @@ See :issue:`100597`
 Description
 ===========
 
-The methods :php:`TYPO3\CMS\Backend\Utility\BackendUtility::getThumbnailUrl()`
-and :php:`TYPO3\CMS\Backend\Utility\BackendUtility::getLinkToDataHandlerAction()`
+The methods :php:`\TYPO3\CMS\Backend\Utility\BackendUtility::getThumbnailUrl()`
+and :php:`\TYPO3\CMS\Backend\Utility\BackendUtility::getLinkToDataHandlerAction()`
 have been marked as deprecated.
 
 
@@ -33,9 +33,9 @@ Migration
 =========
 
 Instead of calling :php:`BackendUtility::getThumbnailUrl()`, inject and use
-the :php:`TYPO3\CMS\Core\Resource\ResourceFactory` directly:
+the :php:`\TYPO3\CMS\Core\Resource\ResourceFactory` directly:
 
-.. code-block::
+.. code-block:: php
 
     // before
     $url = BackendUtility::getThumbnailUrl(2004, [
@@ -51,9 +51,9 @@ the :php:`TYPO3\CMS\Core\Resource\ResourceFactory` directly:
         ->getPublicUrl();
 
 Instead of calling :php:`BackendUtility::getLinkToDataHandlerAction()`, inject
-and use the :php:`TYPO3\CMS\Backend\Routing\UriBuilder` directly:
+and use the :php:`\TYPO3\CMS\Backend\Routing\UriBuilder` directly:
 
-.. code-block::
+..  code-block:: php
 
     // before
     $url = BackendUtility::getLinkToDataHandlerAction(
@@ -77,8 +77,8 @@ and use the :php:`TYPO3\CMS\Backend\Routing\UriBuilder` directly:
     );
 
 In case the second paramter `$redirectUrl` was omitted,
-:php:`getLinkToDataHandlerAction` automatically used the current request uri
-as the return url. In case you relied on this, make sure the `redirect`
+:php:`getLinkToDataHandlerAction` automatically used the current request URI
+as the return URL. In case you relied on this, make sure the `redirect`
 parameter is set to :php:`$request->getAttribute('normalizedParams')->getRequestUri()`.
 
 .. index:: Backend, PHP-API, FullyScanned, ext:backend
```

</details>

<details>
<summary>12.4/Deprecation-100614-DeprecatePageRendererinlineJavascriptWrapAndInlineCssWrap.rst</summary>

```diff
@@ -19,7 +19,7 @@ shall not be used any longer.
 Impact
 ======
 
-`PageRenderer` specifics concerning rendering XHTML or non-HTML5 content are
+:php:`PageRenderer` specifics concerning rendering XHTML or non-HTML5 content are
 not working any longer in affected installations having custom code extending
 :php:`\TYPO3\CMS\Core\Page\PageRenderer`.
 
@@ -37,7 +37,7 @@ Migration
 
 Avoid using the protected properties :php:`$inlineJavascriptWrap` and
 :php:`$inlineCssWrap`. In case any custom code needs to wrap with inline
-:html:`<script>` or :html:`<style>` tags, the new protected methods
+:html:`<script>` or :html:`<style>` tags, use the new protected methods
 :php:`wrapInlineScript($content)` and :php:`wrapInlineStyle($content)`
 within :php:`\TYPO3\CMS\Core\Page\PageRenderer`.
 
```

</details>

<details>
<summary>12.4/Deprecation-100622-ExtbaseFeatureToggles.rst</summary>

```diff
@@ -11,9 +11,9 @@ See :issue:`100622`
 Description
 ===========
 
-Extbase has an own system for feature toggles next to the core feature
+Extbase has an own system for feature toggles next to the Core feature
 toggle API. It has always been marked as internal, but is used for
-a couple of toggles within the extbase framework.
+a couple of toggles within the Extbase framework.
 
 All toggles and the internal PHP API have been marked as deprecated
 in TYPO3 v12 and should be avoided.
@@ -22,11 +22,11 @@ in TYPO3 v12 and should be avoided.
 Impact
 ======
 
-The PHP API for extbase toggles has always been marked as internal. It will
+The PHP API for Extbase toggles has always been marked as internal. It will
 be removed with TYPO3 v13.
 
 The single toggles can still be used in TYPO3 v12, but their triggered
-functionality will be removed with TYPO3 v13 if set to `1`.
+functionality will be removed with TYPO3 v13, if set to `1`.
 
 
 Affected installations
@@ -39,7 +39,7 @@ extension scanner still finds usages of this method in extensions as weak match.
 
 All feature toggles have been marked as deprecated. Setting one of them to `1` in
 TypoScript will trigger a deprecation level log message, they will stop working with
-TYPO v13.
+TYPO3 v13.
 
 
 Migration
@@ -55,7 +55,7 @@ skipDefaultArguments = 1
 
 This is an ancient toggle that was used before routing has been added with
 TYPO3 v9. It allowed to skip the `controller` and `action` argument in frontend
-plugin links, when linking to the default extbase controller / action combination.
+plugin links, when linking to the default Extbase controller / action combination.
 This toggle has been documented as being broken in combination with
 :ref:`Extbase plugin enhancer <t3coreapi:routing-extbase-plugin-enhancer>` already.
 Consuming instances should switch to proper routing configuration instead.
@@ -65,16 +65,16 @@ ignoreAllEnableFieldsInBe = 1
 
 This is another ancient toggle that triggers
 :php:`\TYPO3\CMS\Extbase\Persistence\Generic\Typo3QuerySettings->setIgnoreEnableFields(true)`
-for extbase repositories when used in backend scope. It allows ignoring default :php:`TCA`
+for Extbase repositories when used in backend scope. It allows ignoring default :php:`TCA`
 flags like suppressing of deleted records in queries.
 
-Extbase based backend modules that rely on this toggle being set to `1` can easily
+Extbase-based backend modules that rely on this toggle being set to `1` can easily
 migrate this: When the repository in question is only used in backend context, the
 code below should trigger the same behavior. Note as with other query settings,
 this toggle needs to be used with care, otherwise backend users may see records
 they are not supposed to see.
 
-.. code-block::php
+..  code-block::php
 
     /**
      * Overwrite createQuery to not respect enable fields.
@@ -88,16 +88,15 @@ they are not supposed to see.
 
 When the repository is used in both backend and frontend context, the code
 should be refactored a bit towards a public method that can be set by the
-extbase backend controller only.
+Extbase backend controller only.
 
 enableNamespacedArgumentsForBackend = 1
 ---------------------------------------
 
-This toggle has been introduced in TYPO3 v12. See
-:ref:`changelog-Feature-97096-Non-namespacedArgumentsInExtbaseBackendModules`
+This toggle has been introduced in TYPO3 v12. See :ref:`feature-97096`
 for more details. Extbase backend modules should no longer expect the
-namespace to be set. It may be necessary to adapt some ajax calls and
-request related argument checks in custom modules.
+namespace to be set. It may be necessary to adapt some Ajax calls and
+request-related argument checks in custom modules.
 
 
 .. index:: PHP-API, TypoScript, PartiallyScanned, ext:extbase
```

</details>

<details>
<summary>12.4/Deprecation-100637-ThirdArgumentContentObjectRenderer-start.rst</summary>

```diff
@@ -14,7 +14,7 @@ Description
 When creating instances of the
 :php:`\TYPO3\CMS\Frontend\ContentObject\ContentObjectRenderer`, the
 third argument :php:`$request` when calling :php:`start()` should not be
-hand over anymore. Instead, :php:`setRequest()` should be used
+handed over anymore. Instead, :php:`setRequest()` should be used
 after creating the object.
 
 
@@ -31,16 +31,16 @@ Affected installations
 Instances with casual extensions are probably not affected by this: Instances
 of :php:`ContentObjectRenderer` are usually set-up framework internally.
 
-Using the third agument on :php:`start()` triggers a deprecation level log
+Using the third argument on :php:`start()` triggers a deprecation level log
 message. The extension scanner will *not* find usages, since the method
-same :php:`start()` is used in different context as well and would lead to
+name :php:`start()` is used in different context as well and would lead to
 too many false positives.
 
 
 Migration
 =========
 
-Ensure request is an instance of :php:`Psr\Http\Message\ServerRequestInterface`,
+Ensure the request is an instance of :php:`Psr\Http\Message\ServerRequestInterface`,
 and call :php:`setRequest()` after instantiation instead of calling
 :php:`start()` with three arguments.
 
```

</details>

<details>
<summary>12.4/Deprecation-100653-DeprecatedSomeMethodsInDebugUtility.rst</summary>

```diff
@@ -11,7 +11,7 @@ See :issue:`100653`
 Description
 ===========
 
-The following methods in :php:`TYPO3\CMS\Core\Utility\DebugUtility` have been
+The following methods in :php:`\TYPO3\CMS\Core\Utility\DebugUtility` have been
 marked as deprecated:
 
 * :php:`debugInPopUpWindow()`
@@ -20,7 +20,7 @@ marked as deprecated:
 
 While :php:`debugRows()` and :php:`printArray()` duplicate already existing
 methods, :php:`debugInPopUpWindow()` is discouraged to use as either external
-debuggers, e.g. xdebug or :php:`TYPO3\CMS\Extbase\Utility\DebuggerUtility` may
+debuggers, e.g. Xdebug or :php:`\TYPO3\CMS\Extbase\Utility\DebuggerUtility` may
 be used instead.
 
 
@@ -48,6 +48,6 @@ the former method directly outputs the contents, which is not the case with
 
 The method :php:`debugInPopUpWindow()` is deprecated without a direct
 replacement, consider using an external debugger or
-:php:`TYPO3\CMS\Extbase\Utility\DebuggerUtility::var_dump()` instead.
+:php:`\TYPO3\CMS\Extbase\Utility\DebuggerUtility::var_dump()` instead.
 
 .. index:: PHP-API, FullyScanned, ext:core
```

</details>

<details>
<summary>12.4/Deprecation-100657-TYPO3_CONF_VARSBElanguageDebug.rst</summary>

```diff
@@ -15,7 +15,7 @@ The configuration option :php:`$GLOBALS['TYPO3_CONF_VARS']['BE']['languageDebug'
 has been marked as deprecated in TYPO3 v12, it will be removed with TYPO3 v13
 along with the property :php:`\TYPO3\CMS\Core\Localization->debugKey`.
 
-Setting configuration option `languageDebug` to true adds the label name
+Setting the configuration option `languageDebug` to true adds the label name
 including the path to the :file:`.xlf` file to the output in the backend.
 
 The intention was to allow translators to see where a specific localized
@@ -29,7 +29,7 @@ a benefit at all.
 
 TYPO3 v12 cleaned up lots of label usages and makes them more unique.
 Translators should find single label usages much more easily by searching
-the code base for label names and label files. Also, many fluid templates are
+the code base for label names and label files. Also, many Fluid templates are
 located more transparently and are easier to find, localizing labels within
 PHP classes is also improving a lot. Translators should in general have
 less headaches to see where labels are used, and this will improve further.
@@ -38,7 +38,7 @@ less headaches to see where labels are used, and this will improve further.
 Impact
 ======
 
-The option has been marked as deprecated in TYPO3 v12 and not have any
+The option has been marked as deprecated in TYPO3 v12 and does not have any
 effect anymore with TYPO3 v13.
 
 
```

</details>

<details>
<summary>12.4/Deprecation-100662-ConfigurationManager-getContentObject.rst</summary>

```diff
@@ -11,7 +11,7 @@ See :issue:`100662`
 Description
 ===========
 
-The extbase related method
+The Extbase-related method
 :php:`\TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface->getContentObject()`
 has been marked as deprecated in TYPO3 v12 and should not be used anymore.
 
@@ -26,7 +26,7 @@ from the interface together with their implementations with TYPO3 v13.
 Affected installations
 ======================
 
-Instances with extbase extensions that use :php:`getContentObject()` on
+Instances with Extbase extensions that use :php:`getContentObject()` on
 injected :php:`ConfigurationManager` instances are affected. The extension
 scanner has not been configured to find these calls, since the method
 name is used in different scope as well and would trigger too many
@@ -36,12 +36,12 @@ false positives.
 Migration
 =========
 
-There may be instances with extbase controllers that need to retrieve
-data from the current content object that initiated the frontend extbase
+There may be instances with Extbase controllers that need to retrieve
+data from the current content object that initiated the frontend Extbase
 plugin call.
 
 In this case, controllers can access the current content object from the
-extbase request object using :php:`$request->getAttribute('currentContentObject')`
+Extbase request object using :php:`$request->getAttribute('currentContentObject')`
 instead.
 
 
```

</details>

<details>
<summary>12.4/Deprecation-100670-DIAwareFormEngineNodes.rst</summary>

```diff
@@ -3,7 +3,7 @@
 .. _deprecation-100670-1681916011:
 
 ================================================
-Deprecation: #100670 - DI aware FormEngine nodes
+Deprecation: #100670 - DI-aware FormEngine nodes
 ================================================
 
 See :issue:`100670`
@@ -12,11 +12,11 @@ Description
 ===========
 
 When the FormEngine construct (used when editing records in the backend) has
-been rewritten back in TYPO3 v7, dependency injection for non-extbase
+been rewritten back in TYPO3 v7, dependency injection for non-Extbase
 constructs has not been a thing, yet.
 
-With dependency injection being part of the TYPO3 core extension since TYPO3 v10,
-and the extbase solution being out-phased, it is time to make FormEngine
+With dependency injection being part of the TYPO3 Core extension since TYPO3 v10,
+and the Extbase solution being out-phased, it is time to make FormEngine
 dependency injection aware as well.
 
 This has some impact on classes implementing
@@ -63,7 +63,7 @@ just keep their implementation as is.
 Compatibility with TYPO3 v12 and v13
 ------------------------------------
 
-Extensions that strive for both TYPO3 v12 compatibility, skipping v11, that
+Extensions that strive for TYPO3 v12 compatibility, skipping v11, that
 want to support v13 as well, must adapt their implementations.
 
 As main change, :php:`NodeInterface` no longer declares :php:`__construct()`,
@@ -75,7 +75,7 @@ or if :php:`setData()` should be called after object instantiation. Note
 
 A class with both TYPO3 v12 and v13 compatibility should look like this:
 
-.. code-block:: php
+..  code-block:: php
 
     public function __construct(
         // If the class creates sub elements
@@ -103,7 +103,7 @@ A class with both TYPO3 v12 and v13 compatibility should look like this:
 The class has to be registered for public DI in :file:`Services.yaml` as well, since
 it is instantiated by :php:`NodeFactory` using :php:`GeneralUtility::makeInstance()`:
 
-.. code-block:: yaml
+..  code-block:: yaml
 
     MyVendor\MyExtension\Form\Element\MyElementClass:
       public: true
@@ -123,7 +123,7 @@ so a :yaml:`public: true` entry in :file:`Services.yaml` can be skipped.
 
 A typical class extending :php:`AbstractNode` looks like this:
 
-.. code-block:: php
+..  code-block:: php
 
     public function __construct(
         private readonly NodeFactory $nodeFactory,
```

</details>

<details>
<summary>12.4/Deprecation-100721-LabelRelatedMethodsAndArguments.rst</summary>

```diff
@@ -11,30 +11,30 @@ See :issue:`100721`
 Description
 ===========
 
-The method :php:`TYPO3\CMS\Core\Localization\LanguageService->getLL()` has been
+The method :php:`\TYPO3\CMS\Core\Localization\LanguageService->getLL()` has been
 marked as deprecated.
 
 Along with the deprecation the method
-:php:`TYPO3\CMS\Core\Localization\LanguageService->includeLLFile()` has been
-marked as internal, as it is still used in TYPO3 core for backwards-compatibility
+:php:`\TYPO3\CMS\Core\Localization\LanguageService->includeLLFile()` has been
+marked as internal, as it is still used in TYPO3 Core for backwards-compatibility
 internally, but not part of TYPO3's Core API anymore.
 
-With the introduction of Locales, it is also now not recommended anymore to use
+With the introduction of :ref:`Locales <feature-99694-1674552209>`, it is also now not recommended anymore to use
 custom alternative language keys.
 
 For this reason the argument "alternativeLanguageKeys" of the
-:html:`<f:translate>` ViewHelper  has been deprecated as well, along with the
+:html:`<f:translate>` ViewHelper has been deprecated as well, along with the
 method argument of the same name in
-:php:`TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate()`.
+:php:`\TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate()`.
 
 
 Impact
 ======
 
-Calling the method :php:`TYPO3\CMS\Core\Localization\LanguageService->getLL()`
+Calling the method :php:`\TYPO3\CMS\Core\Localization\LanguageService->getLL()`
 will trigger a PHP deprecation warning.
 
-Calling :php:`TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate()` with
+Calling :php:`\TYPO3\CMS\Extbase\Utility\LocalizationUtility::translate()` with
 the argument "alternativeLanguageKeys" will also trigger a PHP deprecation warning,
 which is the underlying deprecation warning when using the argument
 "alternativeLanguageKeys" of the :html:`<f:translate>` ViewHelper.
@@ -43,30 +43,30 @@ which is the underlying deprecation warning when using the argument
 Affected installations
 ======================
 
-TYPO3 installations within Backend modules using the method :php:`getLL()` or
+TYPO3 installations within backend modules using the method :php:`getLL()` or
 extensions or templates using the translate methods.
 
 The former usually happens in extensions which have been migrated from older
-TYPO3 versions with legacy functionality in Backend modules along
-with :php:`$GLOBALS['LANG']` as LanguageService object.
+TYPO3 versions with legacy functionality in backend modules along
+with :php:`$GLOBALS['LANG']` as :php:`LanguageService` object.
 
 
 Migration
 =========
 
 It is highly recommended to use the full path to a label file along
-with the :php:`sL()` method of :php:`TYPO3\CMS\Core\Localization\LanguageService`:
+with the :php:`sL()` method of :php:`\TYPO3\CMS\Core\Localization\LanguageService`:
 
 Before:
 
-.. code-block:: php
+..  code-block:: php
 
     $GLOBALS['LANG']->includeLLfile('EXT:my_extension/Resources/Private/Language/db.xlf');
     $label = htmlspecialchars($GLOBALS['LANG']->getLL('my_label'));
 
 After:
 
-.. code-block:: php
+..  code-block:: php
 
     $label = $GLOBALS['LANG']->sL('LLL:EXT:my_extension/Resources/Private/Language/db.xlf:my_label');
     $label = htmlspecialchars($label);
```

</details>

<details>
<summary>12.4/Deprecation-98093-Ext_iconAsExtensionIconFileLocation.rst</summary>

```diff
@@ -11,12 +11,12 @@ See :issue:`98093`
 Description
 ===========
 
-Since :issue:`#77349` it is possible to place the extension icon, which is
+Since :issue:`77349` it is possible to place the extension icon, which is
 displayed at various places in the backend, e.g. in the extension manager, in
-an extensions' `Resources/Public/Icons/` directory. The `Resources/` directory
+an extension's :file:`Resources/Public/Icons/` directory. The :file:`Resources/` directory
 is :ref:`by convention <t3coreapi:extension-files-locations>` the place to
-store such files. To simplify extension registration and to fully follow the
-convention have the following file locations been deprecated:
+store such files. To simplify the extension registration and to fully follow the
+convention the following file locations have been deprecated:
 
 * :file:`ext_icon.png`
 * :file:`ext_icon.svg`
@@ -38,7 +38,7 @@ TYPO3 installations with custom extensions using the deprecated file locations.
 Migration
 =========
 
-Place your extension icon as `Extension.*` into `Resources/Public/Icons/`,
+Place your extension icon as :file:`Extension.*` into :file:`Resources/Public/Icons/`,
 as described in :ref:`Feature: #77349 - Additional locations for extension icons <feature-77349>`.
 
 .. index:: Backend, NotScanned, ext:core
```

</details>

<details>
<summary>12.4/Deprecation-99237-MagicImageService.rst</summary>

```diff
@@ -11,14 +11,14 @@ See :issue:`99237`
 Description
 ===========
 
-The class :php:`TYPO3\CMS\Core\Resource\Service\MagicImageService`, which was
+The class :php:`\TYPO3\CMS\Core\Resource\Service\MagicImageService`, which was
 previously used for inline images by EXT:rtehtmlarea has been marked as
 deprecated, since its functionality is no longer needed for `CKeditor`.
 
 Impact
 ======
 
-Using :php:`TYPO3\CMS\Core\Resource\Service\MagicImageService` or one of its
+Using :php:`\TYPO3\CMS\Core\Resource\Service\MagicImageService` or one of its
 public methods will raise a deprecation level log message.
 
 
```

</details>

<details>
<summary>12.4/Important-100207-LetDataMappercreateEmptyObjectUseDoctrineinstantiator.rst</summary>

```diff
@@ -16,7 +16,7 @@ Introduction
 
 This document explains the intended way in which the Extbase ORM thaws/hydrates objects.
 
-Hydrating Objects
+Hydrating objects
 -----------------
 
 Hydrating (the term originates from `doctrine/orm`), or in Extbase terms thawing, is
@@ -25,15 +25,15 @@ is the :php:`DataMapper`. During the process of hydrating, the :php:`DataMapper`
 objects to map the raw database data onto.
 
 Before diving into the framework internals, let's take a look at models from the
-users perspective.
+user's perspective.
 
-Creating Objects with Constructor Arguments
+Creating objects with constructor arguments
 -------------------------------------------
 
-Imagine you have a table `tx_extension_domain_model_blog` and a corresponding model
+Imagine you have a table :sql:`tx_extension_domain_model_blog` and a corresponding model
 or entity (entity is used as a synonym here) :php:`Vendor\Extension\Domain\Model\Blog`.
 
-Now, also imagine there is a domain rule which states, that all Blogs must have a
+Now, also imagine there is a domain rule which states, that all blogs must have a
 title. This rule can easily be followed by letting the blog class have a constructor
 with a required argument :php:`string $title`.
 
@@ -49,15 +49,15 @@ with a required argument :php:`string $title`.
         }
     }
 
-This example also shows how the `posts` property is initialized. It is done in
+This example also shows how the :php:`posts` property is initialized. It is done in
 the constructor because PHP does not allow setting a default value that is of
 type object.
 
-Hydrating Objects with Constructor Arguments
+Hydrating objects with constructor arguments
 --------------------------------------------
 
 Whenever the user creates new blog objects in extension code, the aforementioned
-domain rule is followed. It's also possible to work on the `posts` `ObjectStorage`
+domain rule is followed. It is also possible to work on the :php:`posts` :php:`ObjectStorage`
 without further initialization. :php:`new Blog('title')` is all I need to create
 a blog object with a valid state.
 
@@ -69,12 +69,14 @@ corresponds to what property but only if there is an easy mapping, i.e. if the
 constructor takes argument :php:`string $title` and updates property `title` with it.
 
 To avoid possible errors due to guessing, the :php:`DataMapper` simply
-ignores the constructor at all. It does so with the help of the library `doctrine/instantiator`.
+ignores the constructor at all. It does so with the help of the library `doctrine/instantiator`_.
+
+..  _doctrine/instantiator: https://github.com/doctrine/instantiator
 
 This pretty much explains the title of this document in detail. But there is more
 to all this.
 
-Initializing Objects
+Initializing objects
 --------------------
 
 Have a look at the :php:`$posts` property in the example above. If the :php:`DataMapper`
@@ -128,15 +130,15 @@ method like this:
         }
     }
 
-Mutating Objects
+Mutating objects
 ----------------
 
 I'd like to add a few more words on mutators (setter, adder, etc.). One might think that
 :php:`DataMapper` uses mutators during object hydration but it DOES NOT. `mutators`
 are the only way for the user (developer) to implement business rules besides
 using the constructor.
 
-The :php:`DataMapper` uses `@internal` method :php:`AbstractDomainObject::_setProperty()`
+The :php:`DataMapper` uses the `@internal` method :php:`AbstractDomainObject::_setProperty()`
 to update object properties. This looks a bit dirty and is a way around all business
 rules but that's what the :php:`DataMapper` needs in order to leave the `mutators` to
 the users.
@@ -145,10 +147,10 @@ the users.
 
     While :php:`DataMapper` does not use any mutators, other parts of Extbase do.
     Both, validation and property mapping, either use existing mutators or gather
-    type information from them. This will change in the future but as of TYPO3 12 LTS
+    type information from them. This will change in the future but as of TYPO3 v12 LTS
     this information is correct.
 
-Property Visibility
+Property visibility
 -------------------
 
 One important thing to know is that Extbase needs entity properties to be protected
@@ -158,14 +160,14 @@ private properties of child classes, hence the need to have protected or public
 properties.
 
 
-Dependency Injection
+Dependency injection
 --------------------
 
 Without digging too deep into this topic the following statements have to be made.
 Extbase expects entities to be so called prototypes, i.e. classes that do have a
-different state per instance. DataMapper DOES NOT use Dependency Injection for the
+different state per instance. DataMapper DOES NOT use dependency injection for the
 creation of entities, i.e. it does not query the object container. This also means,
-that Dependency Injection is not possible in entities.
+that dependency injection is not possible in entities.
 
 If you think that your entities need to use/access services, you need to find other
 ways to implement it.
```

</details>

<details>
<summary>12.4/Important-100525-DropUsageOfTextRightAndTextLeft.rst</summary>

```diff
@@ -11,25 +11,25 @@ See :issue:`100525`
 Description
 ===========
 
-The core has dropped support for directional class names to
+The Core has dropped support for directional class names to
 better support RTL languages. We are now preferring the logical
 class names over the directional ones. This change also affects
 the default RTE configuration.
 
-In summary, that means we are dropping the classes .text-right
-and .text-left and replacing them with their logical counterparts
-.text-end and .text-start.
+In summary, that means we are dropping the classes :css:`.text-right`
+and :css:`.text-left` and replacing them with their logical counterparts
+:css:`.text-end` and :css:`.text-start`.
 
-We are still shipping the .text-right and .text-left classes
+We are still shipping the :css:`.text-right` and :css:`.text-left` classes
 with the default RTE content styling. Your content is
 persisted as is and we have no intention of changing this.
 
 You will see the following:
 
 - Your content is still aligned as you set it once
-- The alignment button will not be active anymore for .text-left
-  and .text-right
-- New alignments will now use .text-end and .text-start
+- The alignment button will not be active anymore for :css:`.text-left`
+  and :css:`.text-right`
+- New alignments will now use :css:`.text-end` and :css:`.text-start`
 
 While there is never a good time to introduce such a change,
 we still think this will benefit us all over time.
@@ -48,17 +48,18 @@ CSS for your RTE.
     }
 
 See caniuse for compatibility, which is 96.23% at the time of writing.
-e.g. https://caniuse.com/?search=text-align%3A%20start
+For example: https://caniuse.com/?search=text-align%3A%20start
 
-You need adjust your RTE config if you want to use
+You need to adjust your RTE config, if you want to use
 the old classes.
 
 ..  code-block:: yaml
+    :caption: EXT:my_extension/Configuration/RTE/MyPreset.yaml
 
     editor:
-    config:
+      config:
         alignment:
-        options:
+          options:
             - { name: 'left', className: 'text-left' }
             - { name: 'center', className: 'text-center' }
             - { name: 'right', className: 'text-right' }
```

</details>

<details>
<summary>12.4/Important-100634-Rich-Text-EditorAlwaysEnabledPerUser.rst</summary>

```diff
@@ -3,7 +3,7 @@
 .. _important-100634-1681822129:
 
 =============================================================
-Important: #100634 - Rich-Text-Editor always enabled per user
+Important: #100634 - Rich Text Editor always enabled per user
 =============================================================
 
 See :issue:`100634`
@@ -14,21 +14,24 @@ Description
 Back in TYPO3 v3.x there was an RTE integrated into TYPO3 which only worked
 in Internet Explorer 4+, but not in Mozilla / Firefox browsers. This was a
 huge mess, as not every user / client was able to use an RTE and instead to had
-to write pure HTML in a <textarea> input field with special tags ("typolink" etc).
+to write pure HTML in a :html:`<textarea>` input field with special tags ("typolink" etc).
 
 Since TYPO3 v4 a huge effort were made to integrate HTMLarea as Rich Text Editor,
 which was forked and developed by the TYPO3 community. It was then possible for
 most users working with a real RTE.
 
-In v8, TYPO3 migrated towards CKEditor 4 as a dependency, and CKEditor 5 with
-TYPO3 v12, the Rich-Text-Editor is working very browser-native for modern browsers
+In v8, TYPO3 migrated towards CKEditor 4 as a dependency, and :ref:`CKEditor 5 <feature-96874-1664488673>` with
+TYPO3 v12, the Rich Text Editor is working very browser-native for modern browsers
 without an iframe around the RTE.
 
 A lot of legacy code was moved and migrated, however, one option - the option
-to deactivate the Rich-Text-Editor on a per-user basis - which was necessary in
+to deactivate the Rich Text Editor on a per-user basis - which was necessary in
 TYPO3 v3, has now been removed, as it is not needed in 99.99%
 of TYPO3 installations and users anymore nowadays.
 
-The previous user TSconfig setting `setup.edit_RTE` has no effect anymore.
+Impact
+======
+
+The previous user TSconfig setting :typoscript:`setup.edit_RTE` has no effect anymore.
 
 .. index:: TSConfig, ext:setup
```

</details>

<details>
<summary>12.4/Important-100658-DropUserTSOptionsCreateFoldersInEBAndFolderTreehideCreateFolder.rst</summary>

```diff
@@ -2,16 +2,16 @@
 
 .. _important-100658-1681819486:
 
-==============================================================================================
-Important: #100658 - Drop UserTS options `createFoldersInEB` and `folderTree.hideCreateFolder`
-==============================================================================================
+====================================================================================================
+Important: #100658 - Drop use TSconfig options `createFoldersInEB` and `folderTree.hideCreateFolder`
+====================================================================================================
 
 See :issue:`100658`
 
 Description
 ===========
 
-The UserTS options `createFoldersInEB` and `folderTree.hideCreateFolder` were
+The user TSconfig options :typoscript:`createFoldersInEB` and :typoscript:`folderTree.hideCreateFolder` were
 used in the past to control the existence of the "Create folder" form in Element
 Browser instances. With the migration of the "Create folder" view into a separate
 modal used in EXT:filelist, which is based on Element Browser as well, those
```

</details>

<details>
<summary>12.4/Important-94246-GenericSudoModeConfiguration.rst</summary>

```diff
@@ -2,16 +2,16 @@
 
 .. _important-94246-1681366863:
 
-================================================
-Important: #94246 - GenericSudoModeConfiguration
-================================================
+===================================================
+Important: #94246 - Generic sudo mode configuration
+===================================================
 
 See :issue:`94246`
 
 Description
 ===========
 
-:doc:`Sudo Mode <../9.5.x/Important-92836-IntroduceSudoModeForInstallToolAccessedViaBackend>`
+:doc:`Sudo mode <../9.5.x/Important-92836-IntroduceSudoModeForInstallToolAccessedViaBackend>`
 has been integrated since TYPO3 v9.5.x to protect only Install Tool components. With TYPO3 v12
 it has been changed to a generic configuration for backend routes (and implicitly modules).
 
@@ -25,25 +25,25 @@ All simplified classnames below are located in the namespace :php:`\TYPO3\CMS\Ba
 The low-level request orchestration happens in the middleware :php:`\TYPO3\CMS\Backend\Middleware\SudoModeInterceptor`,
 markup rendering and payload processing in controller :php:`\TYPO3\CMS\Backend\Controller\Security\SudoModeController`.
 
-* a backend route is processed, that requires sudo mode for route URI `/my/route`
-  in :php:`\TYPO3\CMS\Backend\Http\RouteDispatcher`
-* :php:`AccessHandler` tries to find a valid & not expired :php:`AccessGrant` item for the specific
-  :php:`RouteAccessSubject('/my/route')` aspect in the current backend user session data
-* in case no :php:`AccessGrant` can be determined, a new :php:`AccessClaim` is created
-  for the specific :php:`RouteAccessSubject` instance and temporarily persisted in the
-  current user session data - the claim also contains the originally requested route
-  as :php:`ServerRequestInstruction` (a simplified representation of a :php:`ServerRequestInterface`)
-* next, the user is redirected to the user-interface for providing either their own password, or
-  the global install tool password as alternative
-* given, the password was correct, the :php:`AccessClaim` is "converted" to an
-  :php:`AccessGrant`, which is only valid for the specific subject (URI `/my/route`)
-  and for a limited lifetime
+#.  A backend route is processed, that requires sudo mode for route URI `/my/route`
+    in :php:`\TYPO3\CMS\Backend\Http\RouteDispatcher`.
+#.  :php:`AccessHandler` tries to find a valid and not expired :php:`AccessGrant` item for the specific
+    :php:`RouteAccessSubject('/my/route')` aspect in the current backend user session data.
+#.  In case no :php:`AccessGrant` can be determined, a new :php:`AccessClaim` is created
+    for the specific :php:`RouteAccessSubject` instance and temporarily persisted in the
+    current user session data - the claim also contains the originally requested route
+    as :php:`ServerRequestInstruction` (a simplified representation of a :php:`ServerRequestInterface`).
+#.  Next, the user is redirected to the user interface for providing either their own password, or
+    the global install tool password as alternative.
+#.  Given, the password was correct, the :php:`AccessClaim` is "converted" to an
+    :php:`AccessGrant`, which is only valid for the specific subject (URI `/my/route`)
+    and for a limited lifetime.
 
 
 Configuration
 -------------
 
-In general the configuration for a particular route or module looks like this:
+In general, the configuration for a particular route or module looks like this:
 
 .. code-block:: php
 
@@ -56,7 +56,7 @@ In general the configuration for a particular route or module looks like this:
 
 * `group` (optional): if given, grants access to other objects of the same `group`
   without having to verify sudo mode again for a the given lifetime. Example:
-  Admin Tool modules "Maintainance" and "Settings" are configured with the same
+  Admin Tool modules :guilabel:`Maintainance` and :guilabel:`Settings` are configured with the same
   `systemMaintainer` group - having access to one (after sudo mode verification)
   grants access to the other automatically.
 * `lifetime`: enum value of :php:`\TYPO3\CMS\Backend\Security\SudoMode\Access\AccessLifetime`,
```

</details>
